DB/src/main/java/io/deephaven/db/tables/utils/ParquetTools.java

```diff
@@ -438,9 +451,31 @@ public static Table readMultiFileTable(
             allColumns.add(ColumnDefinition.fromGenericType(partitionKey, getUnboxedTypeIfBoxed(partitionValue.getClass()), ColumnDefinition.COLUMNTYPE_PARTITIONING, null));
         }
         allColumns.addAll(schemaInfo.getFirst());
-        return readMultiFileTable(recordingLocationKeyFinder, schemaInfo.getSecond(), new TableDefinition(allColumns));
+        return readPartitionedTable(recordingLocationKeyFinder, schemaInfo.getSecond(), new TableDefinition(allColumns));
+    }
+
+    /**
+     * Reads in a table using metadata files found in the supplied directory.
```

Comment:
Methods now say:
```
     * @see SingleParquetFileLayout
     * @see ParquetMetadataFileLayout
     * @see KeyValuePartitionLayout
     * @see FlatParquetLayout
```
`ParquetMetadataFileLayout` now has:
```
/**
 * <p>{@link TableLocationKeyFinder Location finder} that will examine a parquet metadata file to discover locations.
 *
 * <p>Note that we expect to find the following files:
 * <ul>
 *     <li>{@code _metadata} - A file containing Parquet metadata for all {@link RowGroup row groups} in all
 *     {@code .parquet} files for the entire data set, including schema information non-partitioning columns and
 *     key-value metadata</li>
 *     <li>{@code _common_metadata} <i>(optional)</i> - A file containing Parquet metadata with schema information
 *     that applies to the entire data set, including partitioning columns that are inferred from file paths rather than
 *     explicitly written in {@link org.apache.parquet.format.ColumnChunk column chunks} within {@code .parquet}
 *     files</li>
 * </ul>
 */
```